packages/platforms/accton/x86-64/x86-64-accton-as5916-54x/onlp/PKG.yml

```diff
@@ -0,0 +1,27 @@
+variables:
```

Comment:
@brandonchuang If the r0 revision of this platform will not be supported can you clean this package build to support r1 only? No need to build the package for r0 in this case.

Reply:
@jnealtowns , I just removed r0 part in this file. Please help verify again. Thanks.